### DISCUSSION TITLE
Mortal Kombat Versão Anime

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#428] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/mortal-kombat-anime/

**Comment by one-eyed-willy** [comment]
*Pocket Fighters*, hein?

**Comment by angellisy** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [terça-feira, 31 de maio de 2022](<https://www.vish.blog.br/imagens/mortal-kombat-anime/#comment_1142378419>):

> *Pocket Fighters*, hein?

Denunciou a idade :joy:
